src/analysis/cytoscape/py4cy/cytoscape_util.py

```diff
@@ -0,0 +1,98 @@
+import argparse
+from pathlib import Path, PurePath
+import pathlib
+from typing import List, Dict, Any
+from numpy import append
+import py4cytoscape as p4c
+import time
+import sys
+import os
+from requests.exceptions import RequestException
+import subprocess
+
+SPRAS_ROOT = os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))))
+
+def get_parser() -> argparse.ArgumentParser:
+    '''
+    :return: an argparse ArgumentParser object for parsing command
+        line parameters
+    '''
+    parser = argparse.ArgumentParser(
+        description='Run SPRAS Cytoscape pipeline.')
+
+    parser.add_argument(
+        "--files",
+        action='append'
+    )
+
+    parser.add_argument(
+        "--outdir", 
+        default='output'
+    )
+
+    return parser
+
+def parse_arguments() -> Dict[str, Any]:
+    '''
+    Initialize a parser and use it to parse the command line arguments
+    :return: parsed dictionary of command line arguments
+    '''
+    parser = get_parser()
+    opts = parser.parse_args()
+
+    return opts
+
+def get_pathway_files():
```

Comment:
Is this left over from when it ran as a local demo?

Reply:
yep, I removed this function